<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgSelRange</class>
 <widget class="QDialog" name="DlgSelRange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Выбор диапазона</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>начиная с</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="edStart"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>с шагом</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="edStep"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>в количестве</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="edCount"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>или</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="edPercent"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>% из</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbCount">
         <property name="text">
          <string>XXXX</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>примечание: значение &quot;начиная с&quot; -1 определяет
 последний элемент, -2 предпоследний и т.д.
значение &quot;в количестве&quot;-1 определяет
 все элементы, -2 все без одного и т.д.</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
